sdk/Cargo.toml

```diff
@@ -14,6 +14,7 @@ rust-version = "1.58.0"
 [features]
 async_signer = ["async-trait"]
 file_io = ["openssl"]
+xmp_write = ["xmp_toolkit"]
```

Comment:
Please also add this feature to the README.

Reply:
Added a readme line for the  xmp_write feature